CONTRIBUTING.md

```diff
@@ -80,9 +82,9 @@ Run ` kubectl config current-context` to verify which context you are currently
 
 Run `kubectl config use-context docker-desktop` to switch to a local context. This is the context you will need to run the development environment
 
-1. Navigate to `bootstrap.env` and uncomment the `Local DEV Env` section, ensuring that `DEV=true`
+1. Navigate to `bootstrap.env` and uncomment the `Local DEV Env` section, ensuring that `DEV=true`. Additionally, you can deploy the Secrets Provider locally using HELM. To do so, _also_ uncomment `DEV_HELM`
```

Comment:
... also verify you have installed helm (+ link to helm installation).

wdyt?

Reply:
I added it in the prereqs (: 

> 1. If you intend to deploy the Secrets Provider via Helm, you will need to install the Helm CLI. See [here](https://helm.sh/docs/intro/install/) for instructions on how to do so.